subprojects/snapshots/src/test/groovy/org/gradle/internal/snapshot/impl/DefaultFileSystemMirrorTest.groovy

```diff
@@ -99,37 +88,4 @@ class DefaultFileSystemMirrorTest extends Specification {
         mirror.getMetadata(file.path) == null
         mirror.getSnapshot(file.path) == null
     }
-
-    def "does not discard state about a file that lives in the caches when task outputs are generated"() {
```

Comment:
Why remove this?

Reply:
The `RoutingFileSystemSnapshotter` now takes care of this. And this is where the test went.